### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by respatica**

### CODE REVIEW

**Review by SebastianSilvaDev — CHANGES_REQUESTED** [review]
If this is ready for merge, pls fix merge conflicts with master

File: src/app/dialogs/login/login.component.ts [hunk 1]
```diff
@@ -20,17 +21,18 @@ export class LoginComponent  {
...
   }
-
   
-
-
-
+  get f(){
+    return this.loginForm.controls;
+  }
+
```
Comment (SebastianSilvaDev) [inline src/app/dialogs/login/login.component.ts]: Why f, try a better name, don't know what it refers to

File: src/app/dialogs/login/login.component.html [hunk 2]
```diff
@@ -5,12 +5,35 @@ <h1 mat-dialog-title>Login</h1>
...
+                    class="alert">              
+                    <div *ngIf="f['password'].errors?.['required']">
+                     password is required.
+                    </div>
+                    <div *ngIf="f['password'].errors?.['minlength']">
+                    password in not secure need at least 4 characters.
+                    </div>
+                    <div *ngIf="f['password'].errors?.['forbiddenName']">
+                        password cannot be password.
+                      </div>
```
Comment (SebastianSilvaDev) [inline src/app/dialogs/login/login.component.html]: Use mat-error tag instead

**Review by SebastianSilvaDev — CHANGES_REQUESTED** [review]
Some things are working, couldn't test them all, but some things have to be changed

File: src/app/dialogs/login/login.component.html (hunk 2 above)
Comment (respatica) [reply src/app/dialogs/login/login.component.html]: it change from so many pulls

**Review by SebastianSilvaDev — APPROVED** [review]
LGTM

**Pull request merged**